File: src/main/java/um/tesoreria/facturador/model/dto/ArancelTipoDto.java
package um.tesoreria.facturador.model.dto;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
public class ArancelTipoDto {

    private Integer arancelTipoId;
    private String descripcion = "";
    private Byte medioArancel = 0;
    private Integer arancelTipoIdCompleto;

}

File: src/main/java/um/tesoreria/facturador/kotlin/model/dto/ArancelTipoDto.kt
package um.tesoreria.facturador.kotlin.model.dto

data class ArancelTipoDto(

    var arancelTipoId: Int? = null,
    var descripcion: String = "",
    var medioArancel: Byte = 0,
    var arancelTipoIdCompleto: Int? = null

)
